.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist
+npm-debug.log
```

Comment:
We could keep deleting it, but this way we prevent the file from being committed by mistake.